### PR TITLE
Finish making issue and pr frag dynamic

### PR DESCRIPTION
**Pull request by PhuongPhg**

### CODE REVIEW

**Comment by McSinyx** [comment]
Enlighten me, why would we want to dynamically set styling attributes like margin?  In addition, since data obtain from the network are to not to be set directly on any of the events (source: last lecture), at first glance it's a -1 from me for the added complexity: dynamic code are harder to read (let alone to maintain) than static layout declaration.

My guess would be this is a refactoring effort to allow adding a indefinite number of the same fragment to a parent view, and if that is the case, could you declare the fragment statically and then inflate it upon creation?

**Comment by PhuongPhg** [comment]
> Enlighten me, why would we want to dynamically set styling attributes like margin? In addition, since data obtain from the network are to not to be set directly on any of the events (source: last lecture), at first glance it's a -1 from me for the added complexity: dynamic code are harder to read (let alone to maintain) than static layout declaration.
> 
> My guess would be this is a refactoring effort to allow adding a indefinite number of the same fragment to a parent view, and if that is the case, could you declare the fragment statically and then inflate it upon creation?

It means to create 1 fragment then load it multiple times?

File: app/src/main/java/vn/edu/usth/minigh/PullRequestFragment.java [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+
+            TextView issueTxt = new TextView(this.getContext());
+            issueTxt.setLayoutParams(paramTxt);
+            issueTxt.setTextSize(TypedValue.COMPLEX_UNIT_SP, 18f);
+            issueTxt.setText(prList[i] +" "+ nStatus);
+            issueTxt.setTextColor(getResources().getColor(R.color.secondaryTextColor));
+            llchild.addView(issueTxt);
+
+            ll.addView(llchild);
+            check.addView(ll);
```
Comment (McSinyx) [inline app/src/main/java/vn/edu/usth/minigh/PullRequestFragment.java:140]: I'd phrase it as defining this in XML and inflating it and edit the values that needs to be set dynamically.

**Comment by Huy-Ngo** [comment]
> create 1 fragment then add it multiple times

You can do like I did for RepoPreviewFragment

You can check RecyclerView out, though. It is apparently the proper (or a more proper) way to create a list of similar fragments.

**Review by McSinyx — APPROVED** [review]
I don't see any problem with this.  Blame it on me if merging this set things on fire :smiling_imp:

**Pull request merged**